### PR TITLE
hide ipv6 interfaces by default

### LINKED ISSUE #2849: Make `hide_ipv6_interfaces` on by default

### PR DESCRIPTION
**Pull request by aviramha**
Closes #2849

### CODE REVIEW

File: mirrord-schema.json [hunk 1]
```diff
@@ -790,7 +790,7 @@
         },
         "readlink": {
           "title": "_experimental_ readlink {#experimental-readlink}",
-          "description": "Enables the `readlink` hook.",
+          "description": "DEPRECATED, WILL BE REMOVED",
```
Comment (DmitryDodzin) [inline mirrord-schema.json:793]: This is for ipv6 only right?

**Review by DmitryDodzin — APPROVED** [review]
LGTM, just maybe mention the deprecation in the changelog

File: mirrord/config/src/experimental.rs [hunk 2]
```diff
@@ -40,7 +40,7 @@ pub struct ExperimentalConfig {
     /// ### _experimental_ hide_ipv6_interfaces {#experimental-hide_ipv6_interfaces}
```
Comment (gememma) [inline mirrord/config/src/experimental.rs:40]: should it still be experimental if it's enabled by default?

File: mirrord-schema.json (hunk 1 above)
Comment (aviramha) [reply mirrord-schema.json:793]: no, I just nitpicked a fix (readlink was deprecated but docs stayed same which was weird)

File: mirrord/config/src/experimental.rs (hunk 2 above)
Comment (aviramha) [reply mirrord/config/src/experimental.rs:40]: yes, so users can turn it off if it affects them badly until we roll a fix.

**Pull request merged**